### ISSUE TITLE
Opening empty file with cli opener

### ISSUE THREAD

**Issue by keleset** [jarun/nnn#536]
#### Environment details (Put `x` in the checkbox along with the information)

[x] Terminal Emulator: Alacritty
[x] Shell: bash
[x] Custom desktop opener (if applicable): yes, just a simple script, works flawlessly when used **separately** with empty file OR when used with nnn on **non-empty** file
[x] Program options used: -c

#### Exact steps to reproduce the issue

1) Open any **empty** file with -c and custom cli opener script.
2) Watch message `empty: edit/open with` shown for one second in the bottom (status) line of nnn 
3) Message dissapears, nothing happenes

#### Tl;Dr:

nnn refuses to execute custom cli opener on empty file.

**Comment by jarun** [comment]
As the msg indicates, you can simply press `e` to edit the file.

The check guards against many things:

1. most people don't create text files on *nix as xyz.txt and the average unsuspecting user may not write a custom opener that handles empty files
2. we want to handle known archives internally and someone can create an empty abc.zip and try to trick this

I considered allowing empty files with an extension to be sent to opener, but stopped due to reason 2 above.

It's a minor change. If it's really important to you, just fork and use the tweaked version.
